lib/commands/traffic-converter.ts

```diff
@@ -73,10 +73,12 @@ function requestUriConversion(uri: string, version: string): string {
 }
 
 function processFile(file: string, inputJson: any) {
-  const filePrefix = file.substring(0, file.lastIndexOf("."));
   if (inputJson.Entries !== undefined && inputJson.Entries.length > 0) {
     inputJson.Entries.forEach((entry: ProxyPayload, idx: number) => {
-      let outFile = filePrefix + idx + ".json";
+      let outFile = `${file.substring(0, file.lastIndexOf("."))}_${String(idx).padStart(
```

Comment:
filePrefix is set once out of foreach loop, why do we move it?

Reply:
good catch, fixed.